### PR TITLE
Event details spinner

### PR DESCRIPTION
**Pull request by mxsnyder**

### CODE REVIEW

File: js/apps/events-page/controllers/eventDetailsCtrl.js [hunk 1]
```diff
@@ -2,23 +2,35 @@
...
-		self.data.EventStartDate = '';
-		self.data.EventStartTime = '';
-		self.data.EventEndTime = '';
+		vm.data = {};
+		vm.data.EventStartDate = '';
+		vm.data.EventStartTime = '';
+		vm.data.EventEndTime = '';
+		vm.isLoading = true;
+		vm.isError = false;
+		vm.requestErrorMessage = 'Unfortunately, there was a problem losding this event\'s details. Please try again in a few minutes.';
```
Comment (martypowell) [inline js/apps/events-page/controllers/eventDetailsCtrl.js]: **losding** ...

File: js/apps/events-page/controllers/eventDetailsCtrl.js [hunk 2]
```diff
@@ -2,23 +2,35 @@
...
 
 		const processEventData = (data) => {
-			self.data = data;
-			self.data.EventStartDate = moment(self.data.EventStart).format('MMMM D, YYYY');
-			self.data.EventSchedule = dateUtilityService.formatSchedule(self.data.EventStart, self.data.EventLength, self.data.AllDay);
-			self.isRegistrationRequired = self.data.RegistrationTypeCodeEnum !== 0;
-			self.isOver = moment().isAfter(moment(self.data.EventStart).add(self.data.EventLength, 'm'));
+			vm.data = data;
+			vm.data.EventStartDate = moment(vm.data.EventStart).format('MMMM D, YYYY');
```
Comment (martypowell) [inline js/apps/events-page/controllers/eventDetailsCtrl.js]: I'm assuming we will always have a start date?

File: js/apps/events-page/controllers/eventDetailsCtrl.js (hunk 1 above)
Comment (mxsnyder) [reply js/apps/events-page/controllers/eventDetailsCtrl.js]: D'oh!

File: js/apps/events-page/controllers/eventDetailsCtrl.js (hunk 2 above)
Comment (mxsnyder) [reply js/apps/events-page/controllers/eventDetailsCtrl.js]: That is correct.

**Comment by mxsnyder** [comment]
Typo fixed. Ready for another look.

**Pull request merged**